source/_integrations/oncue.markdown

```diff
@@ -30,3 +30,11 @@ The Oncue by Kohler integration will allow you to monitor the state of your [Onc
 - [KG80R](https://kohlerpower.com/en/residential/generators/product/kg80r)
 
 {% include integrations/config_flow.md %}
+
+## Non-functional and Incorrect Entities
+
+Some entities do not function with some generators.  For example the 20RESA does not report engine coolant temperature, and it will show as 0C (32F).  Also enot related are engine compartment temperature, current, and all related (power) entities.  On the 20RESA at least, the total generated power entry appears bogus.
```

Comment:
```suggestion
Some sensors will report unrealistic values if the generator does not have support for the sensor. For example, the `20RESA` does not report engine coolant temperature, and it will show as 0C (32F). Also unrelated are engine compartment temperature, current, and all related (power) entities. On the `20RESA`, at least, the total generated power may not report a valid value.
```

Reply:
I have a 20resa and get those, but I do have an engine heater installed since I’m in a cold climate.

Reply:
These work on mine.   Also, the total generated power is a crazy large number, but it does seem to increase by a reasonable amount when the generator runs.  

```

          - entity: sensor.sagegen_battery_voltage
            name: Battery Voltage
          - entity: sensor.sagegen_engine_speed
            name: Engine Speed
          - entity: sensor.sagegen_engine_target_speed
            name: Engine Target Speed
          - entity: sensor.sagegen_engine_total_number_of_starts
            name: Engine Total Number Of Starts
          - entity: sensor.sagegen_engine_total_run_time
            name: Engine Total Run Time
          - entity: sensor.sagegen_engine_total_run_time_loaded
            name: Engine Total Run Time Loaded
          - entity: sensor.sagegen_generator_controller_temperature
            name: Generator Controller Temperature
          - entity: sensor.sagegen_generator_controller_total_operation_time
            name: Generator Controller Total Operation Time
          - entity: sensor.sagegen_generator_current_average
            name: Generator Current Average
          - entity: sensor.sagegen_generator_frequency
            name: Generator Frequency
          - entity: sensor.sagegen_generator_true_percent_of_rated_power
            name: Generator True Percent Of Rated Power
          - entity: sensor.sagegen_generator_true_total_power
            name: Generator True Total Power
          - entity: sensor.sagegen_generator_voltage_average_line_to_line
            name: Generator Voltage Average Line To Line
          - entity: sensor.sagegen_latest_firmware
            name: Latest Firmware
          - entity: sensor.sagegen_lube_oil_temperature
            name: Lube Oil Temperature
          - entity: binary_sensor.sagegen_network_connection_established
            name: Network Connection Established
          - entity: sensor.sagegen_source1_voltage_average_line_to_line
            name: Source1 Voltage Average Line To Line
          - entity: sensor.sagegen_source2_voltage_average_line_to_line
            name: Source2 Voltage Average Line To Line

` ` `